test/differentials/thunks.jl

```diff
@@ -35,4 +35,40 @@
             @test stackframe.file == Symbol(@__FILE__)
         end
     end
+
+
+    @testset "Broadcast" begin
+        @testset "Array" begin
+            was_unthunked = 0
```

Comment:
this line's not needed?
```suggestion
```

Reply:
I feel weird about using variables in closures before they are declared.

Reply:
😆

Reply:
do you think it is better to remove it?

Reply:
Yes, but the dont really mind too much either way